tests/system/test_integration.py

```diff
@@ -129,7 +129,7 @@ def check_docs(self, approved, received, doc_type):
                             assert v["version"].startswith(str(v["version_major"]) + ".")
                             continue
 
-                        self.assert_docs(v, appr[k])
+                        self.assert_docs(v, appr.get(k))
```

Comment:
`k` must be in `appr` right (except while fixing tests)?  Could this introduce a situation where `v is None` when it shouldn't be without triggering the test to fail?

Reply:
I changed it because if the test fails, the failure is an exception rather than a diff mismatch.
I actually don't know if that could happen, so I'll change it back

Reply:
Yeah, that's what I meant by "while fixing tests".  Up to you, but we could keep the `get` change and add an additional assertion about `v` directly.